Sources_v3/StreamChatUI/ChatChannelList/ChatChannelListCollectionViewCell.swift

```diff
@@ -12,7 +12,7 @@ open class ChatChannelListCollectionViewCell<ExtraData: ExtraDataTypes>: UIColle
     var uiConfig: UIConfig<ExtraData> = .default
     
     public private(set) lazy var channelView: ChatChannelListItemView<ExtraData> = {
-        let view = uiConfig.channelList.channelListItemView.init(uiConfig: uiConfig)
+        let view = uiConfig.channelList.channelListItemView.init()
         contentView.embed(view)
```

Comment:
nit: I know this is old code, but this should be done in `setUpLayout`

Reply:
Thanks, I rewrote this a bit so it subclasses `CollectionViewCell` Customizable class :)